yt_dlp/extractor/kaltura.py

```diff
@@ -293,6 +303,9 @@ def sign_url(unsigned_url):
                 unsigned_url += '?referrer=%s' % referrer
             return unsigned_url
 
+        if not info:
+            raise UnsupportedError(url)
```

Comment:
example?

Reply:
> > possible fix for #3636
> 
> **Possible** fix?

Yes. Because it prevents from throwing, but I cannot know if the returned formats/metadata work as expected.